April 8-14

```diff
@@ -0,0 +1,8 @@
+April 8 - April 14
+
+Monday: TBA (Gym + Movie Night…?)
```

Comment:
Movie Night: Shazam OR Pet Semetary.

@dead-programmer Add the stats of each of the movies in this comment for other reviewers.

Reply:
@pranmar123 Why don't you go ahead and find those info as a person who "contributes to initial code base"?